src/Sylius/Bundle/ReportBundle/DataFetcher/NumberOfOrdersDataFetcher.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Sylius\Bundle\ReportBundle\DataFetcher;
+
+use Sylius\Bundle\CoreBundle\Doctrine\ORM\OrderRepository;
+use Sylius\Component\Report\DataFetcher\DefaultDataFetchers;
+
+/**
+ * Number of orders data fetcher
+ *
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class NumberOfOrdersDataFetcher extends TimePeriod
```

Comment:
All data fetchers should go to core, cause they depend on core classes. CoreBundle! Go! Go!